.gitattributes

```diff
@@ -15,3 +15,4 @@
 *.dcm filter=lfs diff=lfs merge=lfs -text
 *.zip filter=lfs diff=lfs merge=lfs -text
 *.jpg filter=lfs diff=lfs merge=lfs -text
+*.tiff filter=lfs diff=lfs merge=lfs -text
```

Comment:
For my own knowledge: what this does?

Reply:
This filter makes sure that any `.tiff` files (along with the others specified here) added to the repo will be managed by Git LFS, to avoid bloating the Git history with large binary files.